app/config.ts

```diff
@@ -29,7 +29,8 @@ const fallbackConfig: Config = {
   REDIRECT_TO_GOVERNMENT:
     process.env.REDIRECT_TO_GOVERNMENT === 'true' || false,
   PASSCODE_LENGTH: parseInt(process.env.PASSCODE_LENGTH || '3', 10),
-  DB_PATH: process.env.DB_PATH || './covid_db'
+  DB_PATH: process.env.DB_PATH || './covid_db',
+  THOUSAND_SEPARATOR: process.env.THOUSAND_SEPARATOR || ' '
```

Comment:
Maybe we could use the locale for this instead?

Reply:
Or maybe config is easier :)

Reply:
Yeah, maybe easier. Default is more often `,` I think, but be let's keep it like this. Why not.